farsi_transcribe/audio.py

```
@@ -0,0 +1,253 @@
+"""
+Audio processing module for FarsiTranscribe.
+
+This module handles audio loading, preprocessing, and chunking operations.
+"""
+
+import logging
+from pathlib import Path
+from typing import List, Tuple, Dict, Any, Optional, Iterator
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)